website/blog/2019-12-23-improve-presto-planner.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Improving the Presto planner for better push down and data federation
+author: Yi He
+authorURL: https://www.linkedin.com/in/yi-he-69aa9723/
+---
+**Pintrest:** Yi He
+
+**Alibaba:** Yuan Mei
+
+**Facebook:** James Sun, Maria Basmanova, Rongrong Zhong, Jiexi Lin, Saksham Sachdev, Akshay Pall
```

Comment:
Maybe put "Akshay Pall" to University of Waterloo

Reply:
Done, also put them in alphabetical order.